src/unpacker/tar_repack.py

```diff
@@ -16,15 +16,15 @@ def tar_repack(self, file_path):
 
         archive_directory = TemporaryDirectory(prefix='FAF_tar_repack')
         archive_path = os.path.join(archive_directory.name, 'download.tar.gz')
-        tar_binary = self._repack_extracted_files(extraction_directory, archive_path)
+        tar_binary = self._repack_extracted_files(Path(extraction_directory.name, 'files'), archive_path)
 
         self._cleanup_directories(archive_directory, extraction_directory)
 
         return tar_binary
 
     @staticmethod
-    def _repack_extracted_files(extraction_dir, out_file_path):
-        with Popen('tar -C {} -cvzf {} .'.format(extraction_dir.name, out_file_path), shell=True, stdout=PIPE) as process:
+    def _repack_extracted_files(extraction_dir: Path, out_file_path: str) -> bytes:
+        with Popen('tar -C {} -cvzf {} .'.format(extraction_dir, out_file_path), shell=True, stdout=PIPE) as process:
```

Comment:
we might better use the execute shell command helper function herre

Reply:
but, but, that's not really code I worked on :cry: